docs/dev/types_serialization.md

```diff
@@ -137,7 +137,7 @@ DataStream<WordWithCount> wordCounts = env.fromElements(
     new WordWithCount("hello", 1),
     new WordWithCount("world", 2));
 
-wordCounts.keyBy("word"); // key by field expression "word"
+wordCounts.keyBy(value -> value.word); // .keyBy("word") is @Deprecated
```

Comment:
In my opinion, there is no point in mentioning something that is deprecated. Let's just drop the comment.